CHANGELOG.md

```diff
@@ -5,7 +5,13 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project's packages adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## [v0.4.0] WIP
+## [v0.4.1]
+
+### Changed
```

Comment:
Nit: I think it should be more of `Added` because nothing breaks here.

Reply:
Good point. Done.